enhancements/console/quick-starts.md

```diff
@@ -212,6 +223,27 @@ metadata:
     console.openshift.io/name: explore-serverless
 ```
 
+#### Disabling Quick Starts
+
+1. `operator.openshift.io~v1~Console` CRD provides a `spec.customization` property where an admin will add a property `disabledQuickStarts` with the list of Quick Start Ids to be hidden.
+2. Extend the openshift-console operator so that the console resource data are available in the console ConfigMap and provide them as SERVER_FLAGS.
+3. The Quick Starts listed in `window.SERVER_FLAGS.disabledQuickStarts` will not be visible on the UI
+
+Console operator CRD with disabled Quick Starts:
+```yaml
+apiVersion: operator.openshift.io/v1
+kind: Console
+metadata:
+  name: cluster
+  ...
+spec:
+  customization:
+    brand: online
+    disabledQuickStarts:
+      - 'serverless-application'
```

Comment:
I'm leaning towards creating a specific section for quick starts in case we have some future need.
```suggestion
    quickStarts:
      disabled:
        - 'serverless-application'
```

Reply:
I agree. It makes sense to have a specific section accommodate future quick start related requirements.
Thanks :)
updated the YAML accordingly.